kili/cli.py

```diff
@@ -12,8 +15,86 @@ def cli():
     """Kili Command line interface."""
 
 
+@cli.group(context_settings=CONTEXT_SETTINGS)
+def project():
+    """Commands to interact with a Kili project"""
+
+
+@project.command(name='import')
+@click.argument('files', type=click.Path(), nargs=-1)
+@click.option('--api-key', type=str, envvar='KILI_API_KEY', required=True,
+              help='Your Api Key')
+@click.option('--project-id', type=str, required=True,
+              help='Id of the project to import assets in')
+@click.option('--exclude', '-e', type=click.Path(exists=True), multiple=True,
+              help="Files to exclude from the given files")
+@click.option('--frames', type=bool, default=False, is_flag=True,
+              help="Only for a frame project, import videos as frames. "
+              "The import time is longer with this option")
+@click.option('--fps', type=int,
+              help="Only for a frame project, import videos with this fps")
+#pylint: disable=too-many-arguments
+def import_assets(api_key: str,
+                  project_id: str,
+                  files: Tuple[str, ...],
+                  exclude: Optional[Tuple[str, ...]],
+                  fps: Optional[int],
+                  frames: bool):
+    """
+    Command for adding assets into a project.
+
+    Files can be paths to files or to folders. You can provide several paths separated by spaces.
+
+    Currently, this command does not support:
+
+        - the import of videos from local frames, rich text and time series assets
+
+        - the import of assets with metadata or with a custom external_id
+
+    For such imports, please use the `append_many_to_dataset` method in the Kili SDK.
+    """
+    if files is None:
+        raise ValueError("No files or directory specified.")
+    kili = Kili(api_key=api_key)
+    input_type = kili.projects(project_id,
+                               disable_tqdm=True,
+                               fields=['inputType'])[0]['inputType']
+
+    if input_type != 'FRAME' and (fps is not None or frames is True):
```

Comment:
First treat edge cases, then treat the general case (you're doing the opposite here)

Reply:
That is an edge case, if people try to mention arguments related to a frame project and that it is not

Reply:
It concerns the function structure

```
if (edge_case_1): handle case 1
if (edge_case_2): handle case 2
general case
```

Reply:
`if input_type != 'FRAME' and (fps is not None or frames is True)` is an edge case and raise an early error. The general case is defined later